src/test/java/com/epam/DressesSearch.java

```diff
@@ -0,0 +1,28 @@
+package com.epam;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class DressesSearch {
+
+    @Test
+    @DisplayName("Verify corresponding dress details page showing after search")
+    public void dressesSearch() {
+        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
```

Comment:
since this is the second similar test of yours, you could've extracted the driver instantiation(and closure) into their separate methods.
when using JUnit, preferably as a `@Before` and `@After`  method

Reply:
as a next step the whole driver management could be extracted into a separate package/class/library and also page objects.